src/frontend/src/frontend.rs

```diff
@@ -28,6 +28,7 @@ use crate::service_config::{
 #[serde(default)]
 pub struct FrontendOptions {
     pub mode: Mode,
+    pub node_id: Option<u64>,
```

Comment:
if I understand correctly we are using u64 node id for consistency with datanode. But since frontend is stateless, it can be difficult to generate this unique number from kubernetes operator. Using string here might be more friendly. cc @zyy17

Reply:
@sunng87 agreed. The operator uses K8s deployment to deploy the frontend, and every frontend instance will get the unique hostname like `greptimedb-frontend-7dcc44566b-zp2kj`. @shuiyisong, you can use the hostname(environment variable or use hostname crate) as part of the input of the snowflake algorithm.

Reply:
Maybe `node_id` is not a better naming? It's easy to conflict with the `node_id` in DatanodeOption.

Reply:
@sunng87 @zyy17 PTAL